src/s_inter.c

```diff
@@ -401,19 +396,20 @@ void sys_set_priority(int mode)
 
 void sys_sockerror(char *s)
```

Comment:
maybe the implementation could also go to s_net.h/s_net.c, so we don't have to duplicate it in u_pdsend.c/u_pdreceive.c.

could be something like: 
```
typedef void (*t_printhook)(const char *, ...);

void socket_printerrormsg(t_printhook fn, const char *label, int err);
```

and `sys_sockerr` would become:

```
void sys_sockerror(char *s)
{
    socket_errormsg(&error, s, socket_errno());
}
```

If you want, I can do that.

Reply:
did this in c593faa